shared/constants/tracker2.js

```diff
@@ -132,18 +132,21 @@ export const rpcAssertionToAssertion = (row: RPCTypes.Identify3Row): Types.Asser
     value: row.value,
   })
 
-export const rpcSuggestionToAssertion = (s: RPCTypes.ProofSuggestion): Types.Assertion =>
-  makeAssertion({
-    assertionKey: s.key,
+export const rpcSuggestionToAssertion = (s: RPCTypes.ProofSuggestion): Types.Assertion => {
+  const ourKey = s.key === 'web' ? 'dnsOrGenericWebSite' : s.key
```

Comment:
core tells us a suggestion of 'web' but internally we want to ask the user for a dns or web proof, with one row in the list, so we have this fake item. plumb this through so we get that screen back else dns proofs in the gui are impossible